libraries/core_libs/storage/include/storage/uint_comparator.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <rocksdb/comparator.h>
+
+#include "storage/db_utils.hpp"
+
+namespace taraxa {
+class UintComparator : public rocksdb::Comparator {
+ public:
+  UintComparator() = default;
+  ~UintComparator() = default;
+
+  static const char *kClassName() { return "taraxa.UintComparator"; }
```

Comment:
What is kClassName() for ?

Reply:
I wanted to make same as rocksdb https://github.com/facebook/rocksdb/blob/2e09a54c4fb82e88bcaa3e7cfa8ccbbbbf3635d5/util/comparator.cc#L30